ticktock-compiler/build.gradle

```diff
@@ -17,16 +17,19 @@
 plugins {
   id 'org.jetbrains.kotlin.jvm'
   id 'org.jetbrains.dokka'
-  // TODO apply for uploading to maven central
-//  id 'com.vanniktech.maven.publish'
+  id 'com.vanniktech.maven.publish'
 }
 
-//mavenPublish {
-//  useLegacyMode = false
-//}
+mavenPublish {
+  useLegacyMode = false
+}
```

Comment:
This is the default for the latest release already

Reply:
Ah nice!

```suggestion
```